tesseract_kinematics/kdl/src/kdl_inv_kin_chain_nr.cpp

```diff
@@ -71,9 +75,9 @@ KDLInvKinChainNR::KDLInvKinChainNR(const KDLInvKinChainNR& other) { *this = othe
 KDLInvKinChainNR& KDLInvKinChainNR::operator=(const KDLInvKinChainNR& other)
 {
   kdl_data_ = other.kdl_data_;
-  fk_solver_ = std::make_unique<KDL::ChainFkSolverPos_recursive>(kdl_data_.robot_chain);
-  ik_vel_solver_ = std::make_unique<KDL::ChainIkSolverVel_pinv>(kdl_data_.robot_chain);
-  ik_solver_ = std::make_unique<KDL::ChainIkSolverPos_NR>(kdl_data_.robot_chain, *fk_solver_, *ik_vel_solver_);
+  fk_solver_ = other.fk_solver_;
+  ik_vel_solver_ = other.ik_vel_solver_;
+  ik_solver_ = other.ik_solver_;
```

Comment:
These changes need to be reverted because they cause issues with multi threading. The  solvers need to be unique to each object.